python/src/run_tests.py

```diff
@@ -3,4 +3,5 @@
 
 if __name__ == '__main__':
     suite = unittest.defaultTestLoader.discover('tests')
-    JSONTestRunner(visibility='visible').run(suite)
+    with open('/autograder/results/results.json', 'w') as f:
+        JSONTestRunner(visibility='visible', stream=f).run(suite)
```

Comment:
```suggestion
        JSONTestRunner(visibility='visible', stream=f, buffer=False).run(suite)
```

If you do this, it should prevent showing print statements in the output file. They'll get printed to standard output, which doesn't get buffered, so it never gets shown. The key is to write to a file directly rather than writing the autograder results to standard output, because otherwise it gets jumbled up with other output.